.github/workflows/cli.yaml

```diff
@@ -0,0 +1,134 @@
+name: Karmada CLI Workflow
+on:
+  # Run this workflow every time a new commit pushed to upstream/fork repository.
+  # Run workflow on fork repository will help contributors find and resolve issues before sending a PR.
+  push:
+  pull_request:
+jobs:
+  codegen:
+    name: codegen
```

Comment:
Why run the `codegen` again?

Reply:
I check the steps in `codegen`, it just verifies the codegen, so it's not necessary for this workflow.